index.html

```diff
@@ -23,9 +23,13 @@
 </head>
 <body>
    <section class="body">
-    <section class="sect">
 
-      
+    <section id="popup_mobile" class="multi1"></section>
```

Comment:
[ OPTIONAL ] Cross icon does not show and it's broken on mobile view. Please see the screenshot below and fix that.
![cross](https://user-images.githubusercontent.com/35504149/132014211-91460361-6095-4924-84bf-9f859fe39112.png)

Reply:
okay let me check that.